boards/st/stm32h7b3i_dk/doc/index.rst

```diff
@@ -115,26 +188,48 @@ by the PLL clock at 280MHz. PLL clock is fed by a 24MHz high speed external cloc
 Serial Port
 ===========
 
-The STM32H7B3I Discovery kit has up to 8 UARTs.
-The Zephyr console output is assigned to UART1 which connected to the onboard
-ST-LINK/V3.0. Virtual COM port interface. Default communication settings are
-115200 8N1.
+The STM32H7B3I Discovery kit has up to 10 UARTs. The Zephyr console output is assigned
+to UART1 which connected to the onboard STLINK-V3E. Virtual COM port interface.
+Default communication settings are 115200 8N1.
 
 
 Programming and Debugging
 *************************
 
-See :ref:`build_an_application` for more information about application builds.
-
+Applications for the ``stm32h7b3i_dk`` board configuration can be built and
+flashed in the usual way (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
 
 Flashing
 ========
 
-Flashing operation will depend on the target to be flashed and the SoC
-option bytes configuration.
-It is advised to use `STM32CubeProgrammer`_ to check and update option bytes
-configuration and flash the ``stm32h7b3i_dk`` target.
```

Comment:
```
Flashing may depend on the SoC option bytes configuration, which can be checked and
updated using `STM32CubeProgrammer`_.
```
I think this part should be retained.

Reply:
Dear, @ajarmouni-st, thanks for your review. All requested modifications have been made.